docs/notebooks/inequality_constraints.pct.py

```diff
@@ -100,7 +101,8 @@ def create_bo_model(data):
         }
     )
 
-models = {
+
+initial_models = {
```

Comment:
these will still update (well they won't actually, but they might in the future)

Reply:
any suggestion for now?

Reply:
what you did below was sensible, but i've made it easier with this PR https://github.com/secondmind-labs/trieste/pull/152 ... it would just be `models = map_values(create_bo_model, initial_data)`